### ISSUE TITLE
SA1516 inconsistency with classic StyleCop when assembly-wide attribute is not separated by a blank line from the class definition

### ISSUE THREAD

**Issue by NikolayIT** [DotNetAnalyzers/StyleCopAnalyzers#1923]
For example SA1516 is reported for the following code in the classic StyleCop but not in StyleCopAnalyzers:

``` cs
[assembly: OwinStartupAttribute(typeof(SomeNamespace.Startup))]
namespace SomeNamespace
{
    public class Startup
    {
    }
}
```

I suppose we have to choose between:
- Documenting it and leave it the way it is now
- Fixing it (I will grab it and fix it if we decide this is the way we will solve this)

**Comment by sharwell** [comment]
Does StyleCop Analyzers at least *allow* you to separate these items with a blank line? If so, I think we can just file an issue to fix it for the next (post-1.0) release.

**Comment by NikolayIT** [comment]
If I separate it with blank line StyleCop Analyzers does not report issues.

**Comment by sharwell** [comment]
Cool, we'll just fix it for the next release. :+1:
